GameServerLib/Logic/API/ApiFunctionManager.cs

```diff
@@ -31,7 +31,7 @@ public static byte[] StringToByteArray(string hex)
         internal static void SetGame(Game game)
         {
             _game = game;
-            _logger = Program.ResolveDependency<Logger>();
+            _logger = game.GetLogger();
```

Comment:
For this and all similar future references: This should be a property instead of a function on the Game object, this way the reference can be here as follows:
```CS
_logger = game.Logger;
```
instead of
```CS
_logger = game.GetLogger()
```
it also can be used directly from `_game.Logger` if it makes no sense to store it in a variable anymore